src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -127,9 +126,14 @@ export const TimeSeries = ({
                     fill: '#F00',
                   },
                 },
-              }
-        }
-        baseTheme={theme}
+              },
+          {
+            background: {
+              color: backgroundColor,
+            },
+          },
+        ]}
+        baseTheme={baseTheme}
```

Comment:
This looks to me like it no longer is getting the EUI theme?

Reply:
I don't think TSVB was ever using the EUI charts theme. The original `getTheme` code below only depended on the `@elastic/charts` themes. This is pretty clear from the UI of TSVB compared to charts using EUI, EUI themed charts look much nicer! 😝 

https://github.com/elastic/kibana/blob/e3d01bf450bd42c359ad8ff5607ec1bcf1faa05a/src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/utils/theme.ts#L97-L131

That said I think we could make TSVB use the EUI theme in another PR. It should be easy to merge the `getTheme` logic with the EUI themes.